src/lamplib/src/genny/tasks/genny_runner.py

```diff
@@ -26,6 +28,22 @@ def main_genny_runner(
             raise Exception(f"genny_core not found at {path}.")
         cmd = [path, *genny_args]
 
-        run_command(
-            cmd=cmd, capture=False, check=True, cwd=workspace_root,
-        )
+        # Intercept the workload given to the core binary.
```

Comment:
This is the part that I'm iffy about; it's pretty hacky. We could add more of this logic to click directly, but then the `workload` subcommand would have some commands in cpp and some in python, with different help messages in different locations, etc. 

I think a longer-term solution might be to just remove the cpp CLI parsing completely.

Reply:
I think this is ok. Not much of a difference in this logic living here vs click. 

As you said, the correct solution would be to move the cpp CLI parsing up to click, in which case we can revisit the logic here.